src/ngraph/op/fused/elu.cpp

```diff
@@ -0,0 +1,60 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "ngraph/op/fused/elu.hpp"
+
+#include "ngraph/op/add.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/exp.hpp"
+#include "ngraph/op/maximum.hpp"
+#include "ngraph/op/minimum.hpp"
+#include "ngraph/op/multiply.hpp"
+#include "ngraph/op/subtract.hpp"
+#include "ngraph/op/util/broadcasting.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+op::Elu::Elu(const shared_ptr<Node>& data, const shared_ptr<Node>& alpha)
+    : FusedOp("Elu", {data, alpha})
+{
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::Elu::decompose_op() const
+{
+    auto data = get_argument(0);
+    auto alpha_node = get_argument(1);
+
+    alpha_node = ngraph::op::make_broadcast_node(alpha_node, data->get_shape());
+
+    std::shared_ptr<ngraph::Node> zero_node = std::make_shared<ngraph::op::Constant>(
+        data->get_element_type(), Shape{}, std::vector<double>{0});
+    zero_node = ngraph::op::make_broadcast_node(zero_node, data->get_shape());
+
+    return {std::make_shared<ngraph::op::Maximum>(data, zero_node) +
+            alpha_node * std::make_shared<ngraph::op::Exp>(
+                             std::make_shared<ngraph::op::Minimum>(data, zero_node)) -
+            alpha_node};
+}
+
+shared_ptr<Node> op::Elu::copy_with_new_args(const NodeVector& new_args) const
```

Comment:
Missing `std::` namespace here and before the `make_shared` call.

Reply:
We do not prepend std:: in cpp files that have `using namespace std;`